app/controllers/admin/reports_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Admin::ReportsController < ApplicationController
+
+	def loss_of_pays
+		@loss_of_pays = LossOfPay.includes(:loss_of_pay_infos)
+  end
+
+  def create
+    begin
+      @report = Report.new(params)
+      unless @report.valid?
+        return render action: :loss_of_pays
+      end
+      case params[:report_type].downcase
```

Comment:
When you have used downcase for the report_type, I don't think it'll ever match "when 'lopRefund'" case as 'lopRefund' is in camelCase.